templates_snip/attribute.tpl

```diff
@@ -1,5 +1,3 @@
 /**
- * ${1:{{name}}}
- *
- * @var ${2:{{type}}{{^type}}mixed{{/type}}}
+ * @var ${2:{{type}}{{^type}}mixed{{/type}}} ${1:{{name}}}
```

Comment:
Please leave the default templates as they are. You can overwrite the defaults in your installation through a dedicated template directory.